tests/pact/bad/bad-import-unknown-definition.pact

```diff
@@ -0,0 +1,10 @@
+(module f F
+  (defcap F () true)
+  (defun f () true)
+)
+
+(module g G
+  (use f [g])
+  (defcap G () true)
+  (defun g () (f))
+)
```

Comment:
And obviously no importing definitions that aren't members of the module being imported